docs/osclass.md

```diff
@@ -0,0 +1,123 @@
+<!--[metadata]>
++++
+title = "Getting started with Docker Compose and Osclass"
+description = "Migrating and running an Osclass installation."
+keywords = ["docker, example, package installation, osclass, compose"]
+[menu.main]
+parent="smn_workw_compose"
+weight=6
++++
+<![end-metadata]-->
+
+
+# Getting started with Docker Compose and Osclass
+
+This document teaches you how to migrate an Osclass installation to a Docker
+infrastructure. When you are done, your Docker infrastructure will contain six Docker containers, three  
+server containers and three volume containers:
+
+* Mysql database
+* Postfix server
+* Apache2 Webserver
+* Osclass volume 
+* Database volume
+* Backup volume
+
+This configuration was tested on Osclass 3.5.
+
+## Prerequisites
+
+Before you begin, make sure you have installed the following software:
+
+* VirtualBox
+* Docker
+* Docker Machine 
+* Docker compose 
+
+Refer to the appropriated Docker documentation for different distributions for installing.
+
+## Set up the servers
+
+1. Create the virtualbox machine.
+
+        $ docker-machine create --driver virtualbox dockerenv 
+
+        $ docker-machine active dockerenv
+
+        $ $(docker-machine env dockerenv)
+
+1. Download the scripts using `git clone`.
```

Comment:
Could you change the numbers in the numbered list to the actual numbers? I know it's not a requirement for Markdown, but it's confusing when reading the raw source.